Library/Homebrew/cmd/cleanup.rb

```diff
@@ -57,6 +57,10 @@ def cleanup_keg(keg)
   def cleanup_cache
     return unless HOMEBREW_CACHE.directory?
     HOMEBREW_CACHE.children.each do |path|
+      if path.to_s.end_with? '.incomplete'
```

Comment:
World's smallest nit but should be `".incomplete"`. We've stopped using single quotes unless absolutely necessary, and they shouldn't be necessary here.